pkg/condition/condition.go

```diff
@@ -0,0 +1,149 @@
+package condition
```

Comment:
Please collect the exported methods and internal helpers in separate sections. Makes reading and managing the code easier.